pygmt/tests/test_meca.py

```diff
@@ -5,7 +5,7 @@
 import pandas as pd
 import numpy as np
 import pytest
-from pygmt.helpers import GMTTempFile
+from ..helpers import GMTTempFile
```

Comment:
Since we're on this. Do we want to actually want to continue using relative imports or switch to absolute imports? From https://realpython.com/absolute-vs-relative-python-imports/#pros-and-cons-of-absolute-imports:

> Absolute imports are preferred because they are quite clear and straightforward. It is easy to tell exactly where the imported resource is, just by looking at the statement. Additionally, absolute imports remain valid even if the current location of the import statement changes. In fact, PEP 8 explicitly recommends absolute imports.

> One clear advantage of relative imports is that they are quite succinct. Depending on the current location, they can turn the ridiculously long import statement you saw earlier to something as simple as this:

Reply:
That's a good point. PyGMT doesn't have a deep folder structure, so it seems absolute imports are better. I'm OK with changing relative imports to absolute imports.

Reply:
Please let me know if you want me to convert relative imports to absolute imports in this PR.

Reply:
Yes, I much prefer absolute imports actually. It'll also help a bit with #686. I think it's fine to do it here in this PR.

Reply:
Do you know any tools which can do the conversion automatically？

Reply:
Nope. Probably just regular search and replace.

Reply:
Before that, I think we should let pylint check and warn relative imports.

Reply:
Tried to add either `W0403` or `relative-import` to `disable=` or `enable=` in `.pylintrc`, but none of them warn about relative imports. Am I misunderstanding how pylint works?

Reply:
I see that we have `no-absolute-import` disabled, does that do anything? Actually have no clue how to set this.

Reply:
It's not related:

> no-absolute-import (W1618):
> 
> import missing `from __future__ import absolute_import` Used when an import is not accompanied by from __future__ import absolute_import (default behaviour in Python 3)